index.html

```diff
@@ -6217,6 +6116,159 @@ <h2>Changing the DID controller</h2>
     </section>
   </section>
 
+  <section class="appendix">
+    <h2>Revision History</h2>
+
+    <p>
+This section contains the changes that have been made since the publication of
+this specification as a W3C First Public Working Draft.
+    </p>
+
+    <p>
+Changes since the <a
+href="https://www.w3.org/TR/2021/CR-did-core-20210615/">Second Candidate
+Recommendation</a> include:
+    </p>
+
+    <ul>
+      <li>
+Non-normatively refer to the DID Resolution specification to guide implementers
+toward common DID URL implementation patterns.
+      </li>
+      <li>
+Elaborate upon when DID Documents are understood to start existing.
+      </li>
+      <li>
+Convert PNG diagrams to SVG diagrams.
+      </li>
+      <li>
+Rearrange order of Appendices to improve readability.
+      </li>
+      <li>
+Update the IANA guidance as a result of the IETF Media Type Maintenance
+Working Group efforts.
+      </li>
+      <li>
+Remove at risk feature <code>alsoKnownAs</code> due to insufficient
+implementation experience.
```

Comment:
flagging this line as one that will need to be removed should the `alsoKnownAs` implementers meet the deadline

Reply:
Fixed in 490815df0e5605bbdba9c246b878b35b3b0e1700.